homeassistant/components/zha/cover.py

```diff
@@ -186,6 +186,17 @@ async def async_get_state(self, from_cache=True):
                 self._state = None
 
 
+@MULTI_MATCH(channel_names="tuya_window_covering")
+class ZhaTuyaCover(ZhaCover):
+    """Representation of a ZHA cover."""
+
+    def __init__(self, unique_id, zha_device, channels, **kwargs):
+        """Init this sensor."""
+        super().__init__(unique_id, zha_device, channels, **kwargs)
+        self._cover_channel = self.cluster_channels.get("tuya_window_covering")
+        self._current_position = None
```

Comment:
That wouldn't be needed (already in parent class).
Can you remove and check it?